### PR TITLE
AN0N3 RUINED APRIL 1ST SO NOW I'M PRING THIS FOR REAL

### PR DESCRIPTION
**Pull request by Cheridan**

### CODE REVIEW

**Comment by RemieRichards** [comment]
I hate you.

**Comment by ComicIronic** [comment]
Doesn't this give bloody farts?

**Comment by phil235** [comment]
nice :smile:

**Comment by Cheridan** [comment]
@ComicIronic ONLY IF YOU DRINK BLOOD FIRST

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -287,6 +287,21 @@
 			else
 				..(act)
 
+		if ("fart")
+			if(reagents.total_volume > 0)
+				message = "<B>[src]</B> lets out a juicy one!"
+				reagents.add_reagent("spore", 10) //ew stinky
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/emote.dm]: >not farting all reagents

>not eating clf3 then farting it out

**Comment by Iamgoofball** [comment]
10/10 cheridan

merge posthaste

**Comment by Miauw62** [comment]
dank

**Comment by Jalleo** [comment]
Why did this even happen...

**Pull request merged**